src/components/heatmap/HiatusLegendArc.jsx

```diff
@@ -1,58 +1,61 @@
-        <p className="legend-two legend-before">First Yoshioka</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-three legend-before">Hōzōin</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-four legend-before">Yagyū</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-five legend-before">Baiken</p>
-      </div>
+const ArcBox = styled.div`
+  color: white;
+  width: 200px;
+  height: 32px;
+  text-align: justify;
 
-      <div className="legend-arc-box">
-        <p className="legend-six legend-before">Kojirō</p>
-      </div>
+  @media (max-width: 576px) {
+    width: 180px;
+    font-size: small;
+  }
+`;
 
-      <div className="legend-arc-box">
-        <p className="legend-seven legend-before">Second Yoshioka</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-eight legend-before">Ichijōji's aftermath</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-nine legend-before">Wandering</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-ten legend-before">Farming</p>
-      </div>
-
-      <div className="legend-arc-box">
-        <p className="legend-eleven legend-before">Hosokawa</p>
-      </div>
-      <div className="legend-arc-box">
-        <p className="legend-twelve legend-before">Hiatus Arc</p>
-      </div>
-      <div className="legend-arc-box">
-        <p className="legend-thirteen legend-before">Double Weekly</p>
-      </div>
-    </article>
+const HiatusLegendArc = () => {
+  return (
+    <ArticleTwo>
+      <ArcBox>
+        <StyledP backgroundColor={'#bce784'}>Takezō</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#fff875'}>First Yoshioka</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#28666e'}>Hōzōin</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#06ba63'}>Yagyū</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#508991'}>Baiken</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#7d80da'}>Kojirō</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#ffe156'}>Second Yoshioka</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#a42cd6'}>Ichijōji's aftermath</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#3777ff'}>Wandering</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#4059ad'}>Farming</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#ff6663'}>Hosokawa</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#e3bfc9'}>Hiatus Arc</StyledP>
+      </ArcBox>
+      <ArcBox>
+        <StyledP backgroundColor={'#f4f7f3'}>Double Weekly</StyledP>
+      </ArcBox>
```

Comment:
in this part should i use and object to map in <arcbox /> and <styledp />

Reply:
fixed [commit ](https://github.com/falsepopsky/vagabond-hiatus-chart/pull/29/commits/c15bbe951564a02c771a8260d378676f8154a6db)